Author: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Subject: Remove SSL libs from libs

The Makefile/Configure adds "-lssl -lcrypto" but does not depend libssl-dev
nor does it use any of its functions diretly.
Depending on those should not be needed unless for static compilation thus
removing them.

BTS: https://bugs.debian.org/852924

---
 configure.ac    |    2 +-
 src/Makefile.am |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/configure.ac
+++ b/configure.ac
@@ -3574,7 +3574,7 @@ then
 
 	if test "x$LIBNETAPP_LIBS" = "x"
 	then
-		LIBNETAPP_LIBS="$PTHREAD_LIBS -lxml -ladt -lssl -lm -lcrypto -lz"
+		LIBNETAPP_LIBS="$PTHREAD_LIBS -lxml -ladt -lm -lz"
 	fi
 	AC_MSG_NOTICE([netapp LIBS: $LIBNETAPP_LIBS])
 
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -825,7 +825,7 @@ if BUILD_PLUGIN_NOTIFY_EMAIL
 pkglib_LTLIBRARIES += notify_email.la
 notify_email_la_SOURCES = notify_email.c
 notify_email_la_LDFLAGS = $(PLUGIN_LDFLAGS)
-notify_email_la_LIBADD = -lesmtp -lssl -lcrypto
+notify_email_la_LIBADD = -lesmtp
 endif
 
 if BUILD_PLUGIN_NOTIFY_NAGIOS
